composer.json

```diff
@@ -34,6 +34,7 @@
     ],
     "require": {
         "php": ">=7.2",
+        "ext-json": "*",
```

Comment:
Is this line necessary if we require PHP 7.2+?

https://www.php.net/manual/en/json.installation.php#json.installation

Reply:
Yeah, I was surprised by this! I've never _ever_ used an installation of php without json enabled. 

I added this to satisfy my IDE that complains that `'ext-json' is not listed in composer.json` 

It seems that although ext-json is enabled by default you can still compile php without it with using the `--disable-json` flag, so to use json you should technically announce it as a project dependency. 🤷‍♂️

Reply:
For shizzle, thanks for checking that out